docker/Dockerfile.analysisToolbox

```diff
@@ -0,0 +1,248 @@
+#========================================
+# base-ubuntu
+#========================================
+
+ARG UBUNTU_VERSION=focal
+FROM ubuntu:${UBUNTU_VERSION} AS base-ubuntu
+
+ARG LLVM_VERSION=12
+
+# Install dependencies
+# https://github.com/runtimeverification/evm-semantics/tree/v1.0.1-0e96c8d#ubuntu
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt update \
+    && apt upgrade -y \
+	&& apt install -y \
+		autoconf \
+		bison \
+		clang-${LLVM_VERSION} \
+		cmake \
+		curl \
+		flex \
+		gcc \
+		jq \
+		libboost-test-dev \
+		libcrypto++-dev \
+		libffi-dev \
+		libgflags-dev \
+		libjemalloc-dev \
+		libmpfr-dev \
+		libprocps-dev \
+		libsecp256k1-dev \
+		libssl-dev \
+		libtool \
+		libyaml-dev \
+		lld-${LLVM_VERSION} \
+		llvm-${LLVM_VERSION}-tools \
+		make \
+		maven \
+		netcat-openbsd \
+		openjdk-11-jdk \
+		pkg-config \
+		protobuf-compiler \
+		python3 \
+		python-pygments \
+		rapidjson-dev \
+		time \
+		zlib1g-dev
+
+# Non-duplicates from k repo readme
+# https://github.com/runtimeverification/k/blob/master/package/debian/Dockerfile
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt update                \
+    && apt upgrade -y            \
+    && apt install -y            \
+        build-essential          \
+        debhelper                \
+        gcc                      \
+        libgdbm-dev              \
+        libgmp-dev               \
+        libffi-dev               \
+        libmpfr-dev              \
+        libncurses5-dev          \
+        libnss3-dev              \
+        libreadline-dev          \
+        libsqlite3-dev           \
+        libbz2-dev               \
+        libz3-dev                \
+        parallel                 \
+        pkg-config               \
+        python3-dev              \
+        python3-distutils        \
+        python3-pip              
+
+# Needed for add-apt-repository
+RUN apt install -y \
+        apt-utils \
+	    software-properties-common
+
+# Install git
+RUN add-apt-repository -y ppa:git-core/ppa \
+	&& apt update \
+    && apt upgrade -y \
+	&& apt install -y git
+
+#========================================
+# base-ubuntu-tooling
+#========================================
+
+FROM base-ubuntu AS base-ubuntu-tooling
+
+# Install Haskell
+RUN curl -sSL https://get.haskellstack.org/ | sh
+ENV PATH=/root/.local/bin:$PATH
+
+# Intall Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH=/root/.cargo/bin:$PATH
+
+#========================================
+# base-ubuntu-tooling-z3
+#========================================
+
+FROM base-ubuntu-tooling AS base-ubuntu-tooling-z3
+
+# Install z3
+# Version $Z3_VERSION required by kevm v${KEVM_VERSION}-${KEVM_COMMIT}
```

Comment:
It's now actually more required by solc instead. So need to update these comments.

Reply:
[9188ad8](https://github.com/Byont-Ventures/smart-contract-analysis-tools/pull/38/commits/9188ad8f11aeeaa8b354dd83da3bfb0910f7e02f)